lib/pbench/test/unit/server/test_unpack_tarballs.py

```diff
@@ -1,13 +1,17 @@
-from collections import namedtuple
-import os
+from dataclasses import dataclass
+from logging import Logger
 from pathlib import Path
-from typing import Any
+from typing import Union
 
 import pytest
 
+from pbench.client import JSONOBJECT
 from pbench.common.logger import get_pbench_logger
-from pbench.server.cache_manager import CacheManager, TarballUnpackError
-from pbench.server.unpack_tarballs import UnpackTarballs
+from pbench.server import PbenchServerConfig
+from pbench.server.cache_manager import TarballUnpackError
+from pbench.server.database.models.datasets import Dataset, Metadata
+from pbench.server.sync import Operation
+from pbench.server.unpack_tarballs import Target, UnpackTarballs
```

Comment:
Any interest in adding unit testing for `UnpackTarballs.report()` while you're here?